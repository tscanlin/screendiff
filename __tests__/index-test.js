
const util = require('../util.js')

test('util.getDefaults', () => {
  const defaults = util.getDefaults({})
  expect(defaults).toMatchSnapshot()
});
