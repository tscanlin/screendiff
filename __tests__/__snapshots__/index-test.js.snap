// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util.getDefaults 1`] = `
Object {
  "diff": "screenshots/diff",
  "new": "screenshots/new",
  "original": "screenshots/original",
}
`;
